CensusTracingModule is breaking Bazel build

```
ERROR: /usr/local/google/home/zdapeng/git/grpc-java/core/BUILD.bazel:33:1: Java compilation in rule '//core:internal' failed: Worker process sent response with exit code: 1.
core/src/main/java/io/grpc/internal/CensusTracingModule.java:189: error: cannot find symbol
      eventBuilder.setUncompressedMessageSize(optionalUncompressedSize);
                  ^
  symbol:   method setUncompressedMessageSize(long)
  location: variable eventBuilder of type Builder
core/src/main/java/io/grpc/internal/CensusTracingModule.java:192: error: cannot find symbol
      eventBuilder.setCompressedMessageSize(optionalWireSize);
                  ^
  symbol:   method setCompressedMessageSize(long)
  location: variable eventBuilder of type Builder
```

--- a/core/BUILD.bazel
+++ b/core/BUILD.bazel
@@ -13,6 +13,7 @@ java_library(
         "@com_google_errorprone_error_prone_annotations//jar",
         "@com_google_guava_guava//jar",
         "@com_google_instrumentation_instrumentation_api//jar",
+        "@io_opencensus_opencensus_api//jar",
     ],
 )
 

--- a/repositories.bzl
+++ b/repositories.bzl
@@ -244,6 +244,6 @@ def io_netty_tcnative_boringssl_static():
 def io_opencensus_api():
   native.maven_jar(
       name = "io_opencensus_opencensus_api",
-      artifact = "io.opencensus:opencensus-api:0.5.1",
-      sha1 = "cbd0a716a7d85ac34b83d86b13f0a6655e45c2ba",
+      artifact = "io.opencensus:opencensus-api:0.7.0",
+      sha1 = "bf69d604c0dfcf63356053828a90c3e332acd2b8",
   )

